pyvista/core/dataset.py

```diff
@@ -854,27 +854,7 @@ def get_data_range(self,
         # Use the array range
         return np.nanmin(arr), np.nanmax(arr)
 
-    def points_to_double(self):
```

Comment:
Removing this is out of scope of this PR, but I'm fine with removing this if we're going to include a depreciation error and a placeholder. For example:
```py
def hello_world():
    """Print ``"hello_world"``

    .. deprecated:: 0.4.0
        This function has been deprecated.

    """
    raise DeprecationError('`my_function` has been deprecated')
```